servicex/servicex.py

```diff
@@ -239,7 +246,23 @@ def get_data(selection_query: str, datasets: Union[str, List[str]],
     Returns:
         df                  Pandas DataFrame that contains the resulting flat data.
     '''
-    nest_asyncio.apply()
     loop = asyncio.get_event_loop()
-    return loop.run_until_complete(get_data_async(selection_query, datasets, servicex_endpoint,
-                                   image=image))
+    if not loop.is_running():
```

Comment:
Can you explain in the PR what is the problem you are addressing with asynch-io and an overview of what changed?

Reply:
I've updated the PR's main text. Please check it out and let me know if that makes sense!

Reply:
That is a *great* explanation. PR approval with gold stars!